test/test_pipeline/components/test_setup_image_augmenter.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+from imgaug.augmenters.meta import Augmenter, Sequential
+
+import numpy as np
+
+from autoPyTorch.pipeline.components.setup.augmentation.image.ImageAugmenter import ImageAugmenter
+
+
+class TestImageAugmenter(unittest.TestCase):
```

Comment:
Can you add a test that actually takes an image and apply a transformation, to make sure it properly happen? It can be that it takes a random transformation on a manually created 1,H,W,C numpy array and when applied we make sure some change happened. This will also prevent bugs on theimplementation